stripe.gemspec

```diff
@@ -33,10 +33,13 @@ Gem::Specification.new do |s|
     /\A\.rubocop/,
     /\A\.travis.yml/,
     /\A\.vscode/,
+    %r{\A(bin/tapioca)},
+    /\Asorbet/,
     /\Atest/
   )
   s.files = `git ls-files`.split("\n").reject { |f| ignored.match(f) }
   s.executables   = `git ls-files -- bin/*`.split("\n")
```

Comment:
I think the real change here should be to move executables exposed by this gem to the `exe` folder as per the guidance from Bundler since 2015: https://bundler.io/blog/2015/03/20/moving-bins-to-exe.html

Since the convention is to have binstubs and project-local scripts/executables to live in the `bin` folder, placing executables inside the `exe` folder is a much better approach than filtering specific files.